diff --git a/dist/chunk-S4673CTL.js b/dist/chunk-S4673CTL.js
index 3dd50015738a82b8e95eff162e57a58a1185bc3b..e08bf71b0ac924d64c5538dc56d97a5bf9a46b7e 100644
--- a/dist/chunk-S4673CTL.js
+++ b/dist/chunk-S4673CTL.js
@@ -26,6 +26,7 @@ function render(Component, {
     unmount: () => wrapper.unmount(),
     emitted: (name) => wrapper.emitted(name),
     rerender: (props) => wrapper.setProps(props),
+    wrapper,
     ...getElementLocatorSelectors(baseElement)
   };
 }
diff --git a/dist/pure.d.ts b/dist/pure.d.ts
index 9a8ad6da04e4cecc004b5df265399f6f61497de2..bad4f39a21e089d3695f0dd4ce1f257f8994f23d 100644
--- a/dist/pure.d.ts
+++ b/dist/pure.d.ts
@@ -13,6 +13,7 @@ interface RenderResult<Props> extends LocatorSelectors {
     unmount(): void;
     emitted<T = unknown>(): Record<string, T[]>;
     emitted<T = unknown[]>(eventName: string): undefined | T[];
+    wrapper: any;
     rerender(props: Partial<Props>): void;
 }
 interface ComponentRenderOptions<C, P extends ComponentProps<C>> extends ComponentMountingOptions<C, P> {
